ui/src/MobileClientBuildList.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { ListGroup, ListGroupItem, Icon, Row, Col } from 'patternfly-react';
+import './styles/MobileClientBuild.css';
+
+export class MobileClientBuildList extends Component {
+    render = () => {
+        const mobileClientBuilds = Object.values(this.props.mobileClientBuilds || {});
+        return ( 
+        <div className="text-left">
+            <h6 className="list-group-heading">
+                MOBILE BUILDS
+            </h6>
+            <ListGroup className="list-group-build">
+                    {mobileClientBuilds.map(
+                        ( mobileClientBuild, index ) => (
+                        <ListGroupItem key={index}>
+                            <Row>
+                                <Col md={6} className="text-left list-group-build-item">
+                                        <a href={mobileClientBuild.metadata.selfLink}>{mobileClientBuild.status.config.name} </a>
+                                </Col>
+                                <Col md={6} className="list-group-build-item-description">
+                                <p className="build-info"> 
+                                        {(mobileClientBuild.status.phase === "Running") ?
```

Comment:
We need something similar on https://redhat.invisionapp.com/share/GRJJGUHNSET#/screens/295058245 screen.
So it might make sense to make this a component. Happy for this to be a follow up issue though.

Reply:
I might leave this as it is for now and add a note to this ticket: https://issues.jboss.org/browse/AEROGEAR-7712 to look at making this a reusable component. For now I have cleaned up the code a little bit to make it easier to read.